envi/archs/arm/disasm.py

```diff
@@ -1548,9 +1565,9 @@ def __repr__(self):
                 mnem += 'd'
             if self.iflags & IF_B:
                 mnem += 'b'
-            if self.iflags & IF_H:
+            if self.iflags & IF_H: 
                 mnem += 'h'
-            elif self.iflags & IF_T:
+            if self.iflags & IF_T: #removed el
```

Comment:
From what I read unless mistaken STR is same as STRT but the 'T' variant will run with user mode permissions. Same for STRH and STRHT. The ONLY issue I saw was that there are a couple literal versions of LDR statements and not all of them supported the T variant as commented in the code before this code. I looked through the reference to find the T variants and how they compared. I can double check if necessary but that is what my impression was. And this version of the code matches all of the tests.